sdk/communication/communication-sms/samples-dev/sendSms.ts

```diff
@@ -5,23 +5,35 @@
  * @summary Send an SMS message to 1 or more recipients
  */
 
-import { SmsClient } from "@azure/communication-sms";
+import { SmsClient, SmsSendRequest } from "@azure/communication-sms";
 
 // Load the .env file if it exists
 import * as dotenv from "dotenv";
 dotenv.config();
 
 export async function main() {
+  console.log("== Send SMS Message ==");
+
+  // You will need to set this environment variable or edit the following values
   const connectionString =
-    process.env["COMMUNICATION_CONNECTION_STRING"] ||
+    process.env.COMMUNICATION_SAMPLES_CONNECTION_STRING ||
     "endpoint=https://<resource-name>.communication.azure.com/;<access-key>";
+
+  // create new client
   const client = new SmsClient(connectionString);
-  const sendResults = await client.send({
-    // Phone numbers must be in E.164 format
-    from: "<from-phone-number>",
-    to: ["<to-phone-number-1>", "<to-phone-number-2>"],
+
+  // construct send request
+  const sendRequest: SmsSendRequest = {
+    from: process.env.FROM_PHONE_NUMBER || process.env.AZURE_PHONE_NUMBER || "<from-phone-number>",
```

Comment:
is AZURE_PHONE_NUMBER the one used in the livetest pipeline? maybe we should figure out a better way than all these or statements because I think this will confuse the customers

Reply:
@JoshuaLai @0rland0Wats0n how much effort would it be to make a subconfig for samples env vars
so maybe something like:
COMMUNICATION_SAMPLES_CONNECTION_STRING
COMMUNICATION_SAMPLES_ENDPOINT
SAMPLES_PHONE_NUMBER (alternatively we can keep this as AZURE_PHONE_NUMBER and update the sample.env)
SAMPLES_DESTINATION_PHONE_NUMBERS (change the TO_PHONE_NUMBERS in sample.env)